types/lru-cache/index.d.ts

```diff
@@ -190,4 +190,4 @@ declare namespace LRUCache {
     }
 }
 
-export = LRUCache;
+export default LRUCache;
```

Comment:
The module still uses the `export=` syntax, why are you changing it? If you want to use ES6-style imports with this module, you have to activate the `esModuleInterop` compiler option in your `tsconfig.json`. No need to change the typings.